src/psadt-silent.template/templates/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,67 @@
+﻿$ErrorActionPreference = 'Stop' # stop on all errors
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$zipArchive = Join-Path $toolsDir -ChildPath '[[Zip]]'
+$destination = (Join-Path $toolsDir -ChildPath $env:ChocolateyPackageName)
+
+Get-ChocolateyUnzip -FileFullPath $zipArchive -Destination $destination

```

Comment:
Since the zip archive is embedded, it is probably best to include the license and verification template files.